api/app/__init__.py

```diff
@@ -1,23 +1,34 @@
+import logging
 import os
 
 from apispec import APISpec
 from apispec.ext.marshmallow import MarshmallowPlugin
 from app.models.flask_config import DevelopmentConfig
 from app.models.flask_config import ProductionConfig
-from app.routes import hello
 from flask import Flask
 from flask_apispec.extension import FlaskApiSpec
 from flask_cors import CORS
+from flask_login import LoginManager
 from flask_sqlalchemy import SQLAlchemy
 
+logging.basicConfig(
+    filename="logs/api.log", level=logging.INFO, format="%(asctime)s %(message)s"
+)
+logging.info("Init starting...")
+
+
 db = SQLAlchemy()
+login_manager = LoginManager()
 
 
 def create_app():
-    if os.environ.get("environment") == "production":
+    is_production = os.environ.get("ENVIRONMENT") == "PRODUCTION"
+    if is_production:
         config = ProductionConfig()
     else:
         config = DevelopmentConfig()
+        # initialize dev database here?
+
     # create and configure the app
     app = Flask(__name__, instance_relative_config=True)
```

Comment:
Since we could have multiple environments: `production`, `staging`, `development`, `test`, I would suggest making the config determination more flexible than checking `is_production`

In the flask_config file, define a EnvConfig object which is a dict of env name to config:
```python
EnvConfig = {
  'production': ProductionConfig,
  'staging': StagingConfig,
  'development': DevelopmentConfig,
  'test': TestConfig
}
```

```python
# Now in the app/init file, you can do something simple like this
    from app.config import EnvConfig
    config = EnvConfig[os.environ.get("FLASK_ENV")]
    app = Flask(__name__)
    app.config.from_object(config())
```

This way, you can also set the `SQLALCHEMY_DATABASE_URI` and `SQLALCHEMY_TRACK_MODIFICATIONS` in the config file instead of here

Read more about `FLASK_ENV`, not needing to set `DEBUG` to true for the development environment in the config file and other config related stuff here: https://flask.palletsprojects.com/en/1.1.x/config/#development-production